LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -49,7 +49,11 @@
  Any help would be greatly appreciated.
  */
 
+#if defined(LTH_APP_EXTENSIONS)
```

Comment:
Hmm, the only way I managed to make this work is by adding it in the project's settings, under "Preprocessor Macros" (just like `DEBUG=1`).

A `#define` doesn't work, nor does a new "User Setting". Or I'm doing something wrong here.